src/plugins/orbbec/core/orbbec_camera.cpp

```
@@ -0,0 +1,1757 @@
+        if (pipeline_)
+        {
+            try
+            {
+                pipeline_->stop();
+            }
+            catch (const ob::Error& error)
+            {
+                std::cerr << "Orbbec pipeline stop failed: " << error.what() << std::endl;
+            }
+        }
+        drain_video_frames();
```

Cppcheck (2.21.0)

[error] Unhandled exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)